thinc/backends/cupy_ops.py

```diff
@@ -108,15 +108,15 @@ def frobenius_norm(X):
         gradient *= cupy.minimum(threshold, grad_norm) / grad_norm
         return gradient
 
-    def seq2col(self, seq, nW):
+    def seq2col(self, seq, nW, *, lengths=None):
         """Given an (M, N) sequence of vectors, return an (M, N*(nW*2+1)) sequence.
         The new sequence is constructed by concatenating nW preceding and succeeding
         vectors onto each column in the sequence, to extract a window of features.
         """
-        return _custom_kernels.seq2col(seq, nW)
+        return _custom_kernels.seq2col(seq, nW, lengths=lengths)
 
-    def backprop_seq2col(self, dY, nW):
-        return _custom_kernels.backprop_seq2col(dY, nW)
+    def backprop_seq2col(self, dY, nW, *, lengths=None):
+        return _custom_kernels.backprop_seq2col(dY, nW, lengths=lengths)
```

Comment:
The `lengths` of `_custom_kernels.backprop_seq2col` is typed `const int*` -> what happens if this receives a `None` value?

Reply:
But this goes through the Python wrapper of the kernel first, which handles `lengths=None`:

https://github.com/explosion/thinc/blob/237aa14f48b91615aa7aee497468f0229e742bde/thinc/backends/_custom_kernels.py#L146

Reply:
Thanks! I knew I had to missing a part of the logic in my head :-)